src/rs_quark_assets/components/LoadingBar.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="h-4 w-screen">
```

Comment:
suggestion: hide this entire from the accessibility tree for now

```suggestion
  <div class="h-4 w-screen" aria-hidden="true">
```